### ISSUE TITLE
android artwork exceptions

### ISSUE THREAD

**Issue by puckey** [react-native-kit/react-native-track-player#123]
Tracking a few more exceptions from the play store – these are related to the artwork functionality:

```
java.lang.IllegalArgumentException: 
  at android.util.ArrayMap.validate (ArrayMap.java:567)
  at android.os.Parcel.readArrayMapInternal (Parcel.java:2627)
  at android.os.BaseBundle.unparcel (BaseBundle.java:221)
  at android.os.BaseBundle.keySet (BaseBundle.java:322)
  at android.media.MediaMetadata$Builder.<init> (MediaMetadata.java:578)
  at android.media.session.MediaSession.setMetadata (MediaSession.java:416)
  at android.support.v4.media.session.MediaSessionCompatApi21.setMetadata (MediaSessionCompatApi21.java:108)
  at android.support.v4.media.session.MediaSessionCompat$MediaSessionImplApi21.setMetadata (MediaSessionCompat.java:2382)
  at android.support.v4.media.session.MediaSessionCompat.setMetadata (MediaSessionCompat.java:457)
  at guichaguri.trackplayer.metadata.Metadata.updateArtwork (Metadata.java:222)
  at guichaguri.trackplayer.metadata.components.ArtworkLoader.run (ArtworkLoader.java:62)
```

```
java.lang.ArrayIndexOutOfBoundsException: 
  at android.util.ArrayMap.freeArrays (ArrayMap.java:210)
  at android.util.ArrayMap.put (ArrayMap.java:471)
  at android.os.Bundle.putParcelable (Bundle.java:402)
  at android.support.v4.media.MediaMetadataCompat$Builder.putBitmap (MediaMetadataCompat.java:837)
  at guichaguri.trackplayer.metadata.Metadata.updateArtwork (Metadata.java:218)
  at guichaguri.trackplayer.metadata.components.ArtworkLoader.run (ArtworkLoader.java:62)
```

**Comment by thinhvo-groove** [comment]
I also got this issue in my project. is it somehow to solve this issue?

**Comment by dcvz** [comment]
This should be solved with v1.0.0
